api/src/main/java/org/openmrs/module/fhir/api/AppointmentService.java

```diff
@@ -0,0 +1,39 @@
+/*
```

Comment:
FYI for future work, name pull requests starting with the ticket id... for example, "FHIR-185: Did blah blah blah" :)